networkx/algorithms/assortativity/pairs.py

```diff
@@ -48,10 +48,10 @@ def node_attribute_xy(G, attribute, nodes=None):
         if G.is_multigraph():
             for v, keys in nbrsdict.items():
                 vattr = Gnodes[v].get(attribute, None)
-                for k, d in keys.items():
+                for _ in range(len(keys)):
```

Comment:
```suggestion
                for _ in keys:
```
Was the `range(len(` part added for a reason? Maybe readability? Is this suggestions better?

Reply:
this is much better :)